advanced_guides/synonyms.md

```diff
@@ -13,7 +13,7 @@ However, when a sentence is considered the synonym of another word or sentence,
 
 There are several ways to associate words with each other.
 
-## The One-way association
+## A is synonym of B but B is not synonym of A
```

Comment:
```suggestion
## A is a synonym for B but B is not a synonym for A
```

Reply:
https://www.google.com/search?client=firefox-b-d&sxsrf=ACYBGNTywDmBX9Uj4uO29cIg6aNF6oIyqA%3A1578486234863&ei=2skVXoisNJDkU6PFk4AN&q=what+is+a+synonym&oq=what+is+a+synonym&gs_l=psy-ab.3..0i203l10.3474.5676..5982...1.0..0.130.1341.18j2......0....1..gws-wiz.......35i39j35i39i19j0j0i67j0i131.vncM7uml6L4&ved=0ahUKEwiIgb-6__PmAhUQ8hQKHaPiBNAQ4dUDCAo&uact=5

Reply:
https://www.usingenglish.com/forum/threads/162330-synonym-of-for

Reply:
I haven't check all along with the doc, but there are certainly some sentences with this mistake.